src/plugins/dashboard/public/application/utils/use/use_saved_dashboard_instance.ts

```diff
@@ -129,7 +130,9 @@ export const useSavedDashboardInstance = ({
             chrome.recentlyAccessed.add(
               savedDashboard.getFullPath(),
               savedDashboard.title,
-              dashboardIdFromUrl
+              dashboardIdFromUrl,
+              savedDashboard.getOpenSearchType(),
+              moment(Date.now()).format('MM/DD/YYYY HH:mm')
```

Comment:
```suggestion
              Date.now()
```
Can we just pass `Date.now()` to `add` method? I think timestamp is more generic than a formatted string.

Reply:
changed to Date.now()